content/community/organization/policies.html

```diff
@@ -38,13 +38,28 @@ <h4>Coding Guidelines</h4>
 <HR>
 <h2>Communication</h2>
 
+<h4>General rules</h4>
+
+<p>Be nice to other people. Avoid insults and personal attacks.
+In case someone has innapropriate behavior, raise the issue to members of the
+Haiku development team, or at the last resort get in touch with contact@haiku-inc.org .
+</p>
+
+<p>Appropriate action will be taken, which can range from a warning for the user,
+to banning from the forums and mailing lists or removal of commit access for
+developers.</p>
+
 <h4>The Haiku Fourms</h4>
```

Comment:
You can remove this, I think; these rules should apply to the whole project, not just the forums.